packages/react-charting/src/components/StackedBarChart/StackedBarChart.base.tsx

```diff
@@ -317,26 +320,23 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
   }
 
   private _onBarFocus(pointData: number, color: string, point: IChartDataPoint): void {
-    if (
-      this.state.isLegendSelected === false ||
```

Comment:
Can you explain how setting the state everytime helps.

Reply:
Consider this case:

![image](https://user-images.githubusercontent.com/110246001/197258185-b4bf0a42-4fb3-4430-b957-b7cfb3b7d904.png)

Before the change:
When the 3rd (highlighted) bar is focused, the above condition evaluates to true. The setState function is called and the callout becomes visible with details of the 3rd bar. Now when the 4th (unhighlighted) bar is focused, the condition evaluates to false. The setState function is not called and the callout stays visible with details of the previously focused 3rd bar. So the callout's outdated inner text is used for narration.

After the change:
The setState function is called every time a bar is focused. The callout becomes visible when a highlighted bar is focused and hides when an unhighlighted bar is focused. The callout is always updated with details of the focused bar, which results in correct narration.